examples/data_preprocessing/mimic_dataset/prepare_mimic_cxr.ipynb

```diff
@@ -0,0 +1,1520 @@
+    "\n",
+    "Now the labels get replaced by their id. This gives us the final \"input_list\". Since we will need this file multiple times and the runtime is relatively long, it is cached."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "input_list_full = record_report_split_label_list"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "['input_list.lib']"
+      ]
+     },
+     "execution_count": 15,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "#save the whole file\n",
+    "dump(input_list_full, \"input_list.lib\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We will continue working with the input list of the first n rows where the rows including missing values are dropped. Instead of n=1000 we now have n=932"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "932\n"
+     ]
+    }
+   ],
+   "source": [
+    "#open only first n rows\n",
+    "input_list_pd = load(\"input_list.lib\").iloc[:n,:]\n",
+    "#drop nas\n",
+    "input_list = input_list_pd.dropna().to_numpy()\n",
+    "#save new n\n",
+    "n = len(input_list)\n",
+    "print(n)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Creating rules from Chexpert-labler"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We are using the [\"mentions\" from Chexpert-labler](https://github.com/stanfordmlgroup/chexpert-labeler/tree/master/phrases/mention) for building our rules. Therefore, we need to download the .txt files corresponding to each class from github. The file names are the labels we defined above, where all letters are lowercase and instead of whitespaces there are underscores. So first we do some transformations to get a list of our classes. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#download synonym list from chexpert\n",
+    "classes = list(labels)\n",
+    "#lower case\n",
+    "classes = [each_string.lower() for each_string in classes]\n",
+    "#replace whitespace with _\n",
+    "classes = [each_string.replace(\" \", \"_\") for each_string in classes]\n",
+    "labels2ids = {classes[i]:i for i in range(14)}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we create a new folder in our directory and store the txt files there. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#create folder chexpert_rules\n",
```

Comment:
I would switch this cell and the previous one

Reply:
But I need the "classes" which are generated in the previous cell for this one.